staging/src/k8s.io/apiserver/pkg/storage/etcd3/watcher.go

```diff
@@ -250,14 +250,15 @@ func (wc *watchChan) processEvent(wg *sync.WaitGroup) {
 }
 
 func (wc *watchChan) filter(obj runtime.Object) bool {
-	if wc.internalFilter == nil {
+	if wc.internalFilter.Empty() {
```

Comment:
I do not think it is better than it was .

Reply:
The data type has been changed. It is not a pointer any more.